Devrel

prakarsh-spheron opened the pull request:
Added compute documentation  :

- Getting Started 
- Import from Docker hub 
- Select a new template 

Will be attaching videos, once it's done!

prakarsh-spheron committed:
compute getting started page

prakarsh-spheron committed:
changes to the sidebar

prakarsh-spheron committed:
added docker page to compute

prakarsh-spheron committed:
Start a new template page

prakarsh-spheron committed:
Changes to import from Dockerhub page

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+<img src={require('@site/static/img/pickacontainer.png').default} />
+
+## H2 Select Instance 
```
```suggestion
## Select Instance 
```

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+<img src={require('@site/static/img/viewplans.png').default} />
+
+
+## H2 Additional Configuration 
```
```suggestion
## Additional Configuration 
```

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+You can also configure optional features like port policy mapping, adding envoirment varibale and advanced configuration as adding commands and argurments
+
+### H3 Port Policy Mapping 
```
```suggestion
### Port Policy Mapping 
```

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+To test the port policy mapping, you can use a tool such as Telnet or curl to try to connect to the port that you just opened. If the connection is successful, then the port policy mapping is working as expected.
+
+### H3 Envoirment Variables 
```
```suggestion
### Environment Variables 
```

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+You can intiate by inserting `KEY` and `VALUE` pair to the instance. 
+
+### H3 Advaced Configuration 
```
```suggestion
### Advanced Configuration 
```

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+You can add the following `CMD` instruction to a Dockerfile to specify the custom command and arguments which can be easily added to your instace. 
+
+### H3 Heath Checkup 
```
```suggestion
### Heath Checkup 
```

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+<img src={require('@site/static/img/addconfig.png').default} />
+
+## H2 Final Deployment 
```
```suggestion
## Final Deployment 
```

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+Once you have finished configuring your instance, click on the "Deploy" button. This will start the process of creating your instance. The instance will be deployed, it will reflect in progress and then deployed. 
+
+### H3 Status and Logs
```
```suggestion
### Status and Logs
```

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+Instance logs and instance events refer to the log data generated by instances in a computing environment. This data can include information about the instance's system performance, resource utilization, and any events or errors that occur.
+
+Deploy logs refer to the logs generated during the deployment process of an application or system. These logs typically include information about the deployment process, such as the steps taken, the status of each step, and any errors that occur.
```
Add a point that deploy log can be seen in realtime

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+Deploy logs refer to the logs generated during the deployment process of an application or system. These logs typically include information about the deployment process, such as the steps taken, the status of each step, and any errors that occur.
+
+<img src={require('@site/static/img/Inprogresscompute.png').default} />
```
I'm against having these screenshots as we will upgrade the designs for these pages very soon, and then these images will need revision which is an added overhead, imo.

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+The purpose of these logs is to provide insight into the operation and performance of cloud instances and deployments. They can be used for troubleshooting, performance analysis, and auditing purposes.
+
+<img src={require('@site/static/img/deployedcompute.png').default} />
```
same here

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+To access your instance, you can use Deployment URL or SSH. You can find the connection details for your instance in the Spheron instance dashboard.
+
+### H3 Additional Stats 
```
```suggestion
### Additional Stats 
```

rekpero commented on versioned_docs/version-3.0.0-beta/compute/getting-started.md:
```diff
@@ -0,0 +1,54 @@
+slug: /compute/getting-started
+---
+
+### H3 Spheron Compute 
```
```suggestion
### Spheron Compute 
```

rekpero commented on versioned_docs/version-3.0.0-beta/compute/getting-started.md:
```diff
@@ -0,0 +1,54 @@
+
+<img src={require('@site/static/img/compute-1.png').default} />
+
+## 	Add Organization Name and Username 
```
```suggestion
## Create your Organization
```

rekpero replied:
Btw I don't understand the context change here. Before this, we are talking about how to create a cluster but now it changed to creating a new org. I don't think it make a lot of sense to me when I read thru get started doc

rekpero commented on versioned_docs/version-3.0.0-beta/compute/getting-started.md:
```diff
@@ -0,0 +1,54 @@
+
+## 	Add Organization Name and Username 
+
+Add Organisation name and the username aslo you can add organisation avatar and then click on save. 
```
```suggestion
You need to add the Organisation name and the username. Also, you can add an avatar of your liking and then click on Save. 
```

rekpero commented on versioned_docs/version-3.0.0-beta/compute/getting-started.md:
```diff
@@ -0,0 +1,54 @@
+
+Add Organisation name and the username aslo you can add organisation avatar and then click on save. 
+
+<img src={require('@site/static/img/addorg.png').default} />
```
remove this one too

rekpero commented on versioned_docs/version-3.0.0-beta/compute/getting-started.md:
```diff
@@ -0,0 +1,54 @@
+
+## Create a new project 
+
+static/img/createnew.png
```
what's this?

rekpero commented on versioned_docs/version-3.0.0-beta/compute/getting-started.md:
```diff
@@ -0,0 +1,54 @@
+
+static/img/createnew.png
+
+<img src={require('@site/static/img/createnew.png').default} />
```
What is your understanding of this screenshot? Is it enough to pass on a screenshot without giving context over what they need to choose or which option is for which use case?

rekpero commented on versioned_docs/version-3.0.0-beta/compute/getting-started.md:
```diff
@@ -0,0 +1,54 @@
+
+## Create new cluster 
+
+Click to create a new cluster, and the organization name will be reflected on the top left corner. 
```
We need to explain terminology here. What does it mean by Cluster or just give the user a simple step-by-step walkthrough of how to deploy on compute without the requirement?

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+slug: /compute/import-from-docker-hub
+---
+
+## Gettign Started from DockerHub 
```
```suggestion
## Getting Started with DockerHub 
```

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+The steps are followed from the getting started page of compute and branched into the following steps if chose to import the image from docker hub, this is a step-by-step guide to import from dockerhub. 
+
+- Cluster name: Choose a descriptive name for your instance.
+- Repository: Add the repository where you want the instance to be.
```
```suggestion
- Repository: Add the repository where your image is present in Dockerhub.
```

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+- Cluster name: Choose a descriptive name for your instance.
+- Repository: Add the repository where you want the instance to be.
+- Tag: Assign metadata to the instance for easier management and organization.
```
```suggestion
- Tag: Provide a tag of your docker image. It is optional, so you leave it empty. Spheron will assign `latest` if it's left empty.
```

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+- Repository: Add the repository where you want the instance to be.
+- Tag: Assign metadata to the instance for easier management and organization.
+
+<img src={require('@site/static/img/pickacontainer.png').default} />
```
Can you remove this screenshot, looks very weird in the UI

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+## H2 Additional Configuration 
+
+You can also configure optional features like port policy mapping, adding envoirment varibale and advanced configuration as adding commands and argurments
```
```suggestion
You can also configure optional features like port policy mapping, adding environment variables, and advanced configuration as adding commands and arguments.
```

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+### H3 Port Policy Mapping 
+
+Port policy mapping is a feature of some cloud providers that enables you to control the access to the ports of your instances, by defining rules that specify which sources are allowed to access which ports. The main goal of port policy mapping is to improve the security of your instances by limiting the exposure to the network.For example, to allow incoming HTTP traffic from anywhere, you can ad ...
```
```suggestion
Port policy mapping is used in Spheron for users to map their internal ports and expose them. It controls the access to the ports of your instances by defining rules that specify the external port. 
- Users can choose from port 80 (HTTP port). If you choose this, you will receive a reverse DNS link to connect to your instance.
- Users can also choose a random port that will assign an external port from 30000-31000 range. You can connect with the host URL and external port.
```

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+You can monitor the status of your instance creation on the "Instances" page. Once the instance has been created, you can start using it.
+
+<img src={require('@site/static/img/addconfig.png').default} />
```
remove this

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+### H3 Status and Logs
+
+Stats represent Total, Active, Starting, Closed and Failed to Start instances. 
```
```suggestion
Stats represent Total, Active, Starting, Closed, and Failed to Start instances. 
```

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+Stats represent Total, Active, Starting, Closed and Failed to Start instances. 
+
+Instance logs and instance events refer to the log data generated by instances in a computing environment. This data can include information about the instance's system performance, resource utilization, and any events or errors that occur.
```
```suggestion
Instance logs and instance events refer to the log data generated by instances in a computing environment. This data can include information about the instance's system performance, resource utilization, and any events or errors.
```

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+<img src={require('@site/static/img/deployedcompute.png').default} />
+
+To access your instance, you can use Deployment URL or SSH. You can find the connection details for your instance in the Spheron instance dashboard.
```
```suggestion
To access your instance, you can use the instance connection URL. You can find the connection details for your instance in the Spheron instance dashboard.
```

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+## H2 Select Instance 
+
+- Storage: Compute instances typically come with either local (on-instance) or network attached storage options. Local storage is faster but is limited in size and is deleted when the instance is terminated. Network attached storage can be larger and persist after instance termination but can be slower to access.
```
Can you check if this is correct for akash network? Check with @Man-Jain or @dusanstanis

Man-Jain replied:
No, This isn't correct. 

Some points on the Storage 

- We do not support persistent storage ATM.
- Yes, but there are a few considerations you need to keep in mind. First, anything that is in the ephemeral storage will get lost if the provider restarts. This can be countered with persistent storage to an extent. However, you also won't have any commands that runs on start-up. This can be solved by adding all commands you want to run in the SDL or by creating a Docker Image.
- Ephemeral storage is faster (in terms of IOPS) and persistent storage can be slower (in terms of IOPS).  This is due to network latency associated with persistent storage and as the storage nodes (or the pods to storage nodes) are connected over the network.
- if using ephemeral storage, it will be lost on crashes, reboots, or if your deployment is closed.

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+*Note: The specific cost and resource information may vary depending on the cloud provider and region. It is recommended to check the provider's pricing documentation for up-to-date and accurate information.*
+
+<img src={require('@site/static/img/viewplans.png').default} />
```
put this screenshot at the beginning. After the title and before the para start

rekpero commented on versioned_docs/version-3.0.0-beta/compute/import-from-docker-hub.md:
```diff
@@ -0,0 +1,99 @@
+
+Port policy mapping is a feature of some cloud providers that enables you to control the access to the ports of your instances, by defining rules that specify which sources are allowed to access which ports. The main goal of port policy mapping is to improve the security of your instances by limiting the exposure to the network.For example, to allow incoming HTTP traffic from anywhere, you can ad ...
+
+You can select port 80 or Random port mapped to the container port. 
```
you can remove this sentence

rekpero commented on versioned_docs/version-3.0.0-beta/compute/start-from-a-new-template.md:
```diff
@@ -0,0 +1,55 @@
+
+## Start from a new template 
+
+In this segement you can choose from any templated segment, Create a new cluster based on a template, push new code and that's it!
```
I'll comment this page later, lot of mistakes are here

rekpero reviewed (changes requested):
please check my comments

prakarsh-spheron committed:
Changes to the billing category

prakarsh-spheron committed:
- Removed Skynet and Pinata from Decentralized storage, added IPFS
- Removed Minting 101 from quick start
- Removed Super clan category and pages
- Changes to configuring wallet
- Added Deposit Fund
- Withdraw Fund
- Deposit Fund
- Added new coupons section
- Changed the image of usage page
